packages/wonder-blocks-timing/index.js

```diff
@@ -4,6 +4,17 @@ import type {
     IInterval,
     IScheduleActions,
     ITimeout,
+    WithActionScheduler,
```

Comment:
What is the reasoning about only exposing the HOC and keeping the provider private?

Asking this to get more context, as I have been seeing that we tend to use `function-as-children` instead of HOCs.

Reply:
Great question. So, I had some thoughts about this, and in this case, the HOC should provide an API that is MUCH easier to to get right. If you need timeouts, intervals, etc., just wrap with the HOC and you'll have a prop, and your actions will be cleared on unmount.

With the provider, it could be more easily mistakenly misused whereby whatever gets rendered in the children could unmount without the timers being cleared, which is more likely to lead to weird things like accessing elements that are no longer in the DOM and such.

I think the HOC is the safer use case and only having one makes it easier to get it right and consistent. However, I'm open to considering both.

Reply:
Sounds good to me! thanks for the explanation.